### ISSUE TITLE
Make failing on #4791c30 due to no rule to make chain.h

### ISSUE THREAD

**Issue by southamptonjake** [SDL-Hercules-390/hyperion#408]
Making Hyperion 4791c3068846c4b57c90677e04df2a7e0d63e3e9 on ubuntu, but looking at past commits it seems like `chain.h` was removed (https://github.com/SDL-Hercules-390/hyperion/commit/4f0ae744a13587574efda06d04718a0954f4e61b). When I revert to before this commit, everything builds correctly.

```
make[2]: *** No rule to make target 'chain.h', needed by 'all-am'. Stop.
make[2]: *** Waiting for unfinished jobs....
  CC       cardpch.lo
make[2]: Leaving directory '/home/jake/Downloads/hyperion'
make[1]: *** [Makefile:2569: all-recursive] Error 1
make[1]: Leaving directory '/home/jake/Downloads/hyperion'
make: *** [Makefile:1904: all] Error 2
```

**Comment by wrljet** [comment]
Jake,

If you run `./autogen.sh` before `configure`, it will work for you (until the repo is corrected).

Bill

**Comment by Fish-Git** [comment]
_(Oops!)_ &nbsp;Sorry. &nbsp;My bad. &nbsp;`:(`

**Comment by Fish-Git** [comment]
Fixed by commit 7bcdc7e6c2597dc2bac372546a5b808a921d3c50.

Please refresh your re-clone your repository and try again.

Closing.
